### PR TITLE
Updates how to integrate token onramp

### PR DESCRIPTION
**Pull request by joshuanwankwo**

### CODE REVIEW

File: data/guides/how-to-use-token-on-ramp-aggregator.mdx [hunk 1]
```diff
@@ -28,15 +30,11 @@ Base URL:
...
-| `fiat_amount`  | `No`            | `<200>`|
-| `crypto_currency`  | `No`            | `<ETH>`|
-| `network`  | `No`            | `<Ethereum>`|
-| `wallet_address`  | `No`            | `<0x000000>`|
+Now let’s get straight into building! The only thing you need to do is to add
+the base URL to your iframe or webview on mobile, the on-ramp provider will
+guide you on how to complete your purchase.
 
-Once you get the URL, you can open it in your app (e.g. webview, iframe). The on-ramp provider will guide the user to finish the purchase process, including typing in payment information and KYC if needed.
+![token_onramp_aggregator.png](/docs/images/tutorials/token_onramp_aggregator.png)
```
Comment (l2l2l2l) [inline data/guides/how-to-use-token-on-ramp-aggregator.mdx]: Please add another section how to test the integration: 

* onramp-staging.mirrorworld.fun
* test credit cards they can use. Please go to moonpay/transfi/gatefi/onramp.money documentation site, they have test credit card info.

File: data/guides/how-to-use-token-on-ramp-aggregator.mdx [hunk 2]
```diff
@@ -8,17 +8,19 @@ author:
 tags:
   - Token OnRamp
   - Wallet
+  - Fiat
```
Comment (l2l2l2l) [inline data/guides/how-to-use-token-on-ramp-aggregator.mdx]: We have a strict rule on what tags should be used. Please remove this.

File: data/guides/how-to-use-token-on-ramp-aggregator.mdx [hunk 3]
```diff
@@ -28,15 +30,11 @@ Base URL:
 http://onramp.mirrorworld.fun/
 ```
 
-| URL Param  | Required?        | Sample           |
```
Comment (l2l2l2l) [inline data/guides/how-to-use-token-on-ramp-aggregator.mdx:31]: Why removing this table? If removed, how can developer knows what url param they can pass in?  params options: https://mirrorworldfun.notion.site/On-Ramp-Aggregator-c533d732a7c348ddafc7181c66390312?pvs=4 Please include that in article

File: public/images/tutorials/token_onramp_aggregator.png
Comment (l2l2l2l) [inline public/images/tutorials/token_onramp_aggregator.png:1]: Why creating a new file? Where is it used?

**Review by lli839 — APPROVED** [review]

**Pull request merged**